### ISSUE TITLE
chroma: Incorrect syntax highlighting for .diff code

### ISSUE THREAD

**Issue by dmerejkowsky** [gohugoio/hugo#4143]
hugo version: v0.32-DEV-A2D81CE9

# Steps to reproduce

* in config.yml
```yaml

pygmentsStyle: native
pygmentsUseClasses: true

```

* in content/post/hello.md

```markdown
# Hello
{{< highlight diff >}}
- foo
+ bar
{{< /highlight >}}
```
# Actual

Generated html looks like:

```html

<div class="highlight">
...
<span class="err">+ bar</span>
...
</div>
```
Note the 'err' class.

# Expected

No `err` class.

# Notes

I think the problem is that the chroma lexer does not see the last `\n` and thus does not manage to properly parse the contents of the code block.

**Comment by kaushalmodi** [comment]
This needs to be fixed in Chroma; related: https://github.com/alecthomas/chroma/issues/55

**Comment by moorereason** [comment]
This has been fixed for a long time (since Hugo v0.32?).

**Comment by dmerejkowsky** [comment]
Indeed :) The bug was in chroma and has since been fixed.

Sorry I forgot to update this one. Cheers!
